docs/source/recipes/developing-a-project.md

```diff
@@ -1,32 +0,0 @@
----
-myst:
-  html_meta:
-    "description": "Developing a Volto project"
-    "property=og:description": "Developing a Volto project"
-    "property=og:title": "AppExtras component"
-    "keywords": "Volto, Plone, frontend, React, app extra component"
----
-
-(frontend-developing-a-project-label)=
-
-# Developing a project
-
-When you start developing a Plone project, it is recommended that you use Plone's cookiecutter boilerplate project generator.
-See {doc}`plone:install/create-project` for more information.
-
-The generator will output the project folder structure.
-It is organized in three folders named `frontend`, `backend`, and `devops`, each of which corresponds to its primary use.
-It also contains the convenience `Makefile` commands to perform all the usual actions while developing.
-
-## Add-on first approach
```

Comment:
@stevepiercy The info under this heading is a really key concept for Volto (how a project is organized in the filesystem, and the fact that changes should be made in an addon). It needs to be kept somewhere... maybe in an expansion of https://6.docs.plone.org/volto/getting-started/index.html along the lines of "now that you installed Plone, you have a frontend with this structure, here are some next steps..."

Reply:
I have an open related issue where [I proposed to move around bits and pieces](https://github.com/plone/volto/issues/5730#issuecomment-1930949947).

I think this content should go in there, as its intended audience is the same, specifically for people who want to develop a project with Volto.

I would appreciate your feedback on that issue comment in that issue, as I think @tiberiuichim will be tied up with client work for the next month.

Reply:
Moved to `getting-started/roadmap.md`. Will pick this up over in a PR for https://github.com/plone/volto/issues/5730 for more concept shuffling.